### PR TITLE
ISPN-13686 tx cache docs

### PR DESCRIPTION
**Pull request by oraNod**
Docs for transaction caches.

### CODE REVIEW

File: src/app/assets/languages/en.json [hunk 1]
```diff
@@ -77,9 +77,9 @@
     "cache-filter-feature-label" : "Feature",
     "cache-filter-feature-bounded" : "Bounded",
     "cache-filter-feature-indexed" : "Indexed",
-    "cache-filter-feature-persistent" : "Persistent",
-    "cache-filter-feature-trans" : "Transactional",
-    "cache-filter-feature-secure" : "Secured",
+    "cache-filter-feature-persistent" : "Persistence",
+    "cache-filter-feature-trans" : "Transactions",
+    "cache-filter-feature-secure" : "Authorization",
```
Comment (oraNod) [inline src/app/assets/languages/en.json:82]: @dipguptaredhat Hey I tried to change the labels in the feature drop-down but it's still showing "Transactional" and I couldn't quite figure out why this doesn't take effect. Would you mind taking a look please? 

image

File: src/app/assets/languages/en.json [hunk 2]
```diff
@@ -269,20 +269,22 @@
...
           "mode-manual": "MANUAL",
           "mode-auto": "AUTOMATIC",
-          "transactional": "Transactional",
-          "transactional-description": "Allows {{brandname}} to participate in distributed transactions compliant with JTA.",
-          "transactional-mode": "Transactional mode",
-          "transactional-mode-tooltip": "Configure the transactional mode for the cache. Transactional caches are not shared across nodes and are not replicated. Transactional caches are useful for high-performance transactional data stores.",
+          "transactional": "Transactions",
+          "transactional-description": "Configure {{brandname}} to participate in JTA-compliant distributed transactions.",
+          "transactional-mode": "Transaction mode",
+          "transactional-mode-tooltip": "Configure the mode that {{brandname}} uses when carrying out ACID (atomicity, consistency, isolation, durability) transactions to ensure the cache state is consistent.",
```
Comment (oraNod) [inline src/app/assets/languages/en.json:275]: @pruivo Do you think it's too much to put the ACID reference here?

Comment (pruivo) [reply src/app/assets/languages/en.json:275]: probably :) ACID property is not much relevant for transaction mode.

Comment (oraNod) [reply src/app/assets/languages/en.json]: Thanks, I've removed it.

**Review by karesti — APPROVED** [review]

**Pull request merged**